WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -151,6 +151,13 @@ class AccountSettingsService {
         return notificationObserver.map(getSettings).startWith(getSettings())
     }
 
+    func primarySiteNameForSettings(settings: AccountSettings?) -> String {
```

Comment:
Why do you accept an optional AccountSettings here? There's no point in calling this method with a `nil` value